packages/atomic/src/components/common/atomic-focus-trap.tsx

```diff
@@ -34,7 +34,13 @@ export class AtomicFocusTrap {
   private readonly hiddenElements: Element[] = [];
 
   hide(element: Element) {
-    if (element.hasAttribute('aria-hidden')) {
+    // aria-hidden -> already hidden
+    // aria-live or atomic-aria-live -> must not be hidden otherwise it won't announce dynamic changes in the live region
+    if (
```

Comment:
This was discussed on slack: Essentially, the focus trap (when the modal is opened) would add `aria-hidden=true` on the aria live region, preventing it from being announced by the screen reader.